sdk/azcore/runtime/policy_http_trace.go

```diff
@@ -0,0 +1,114 @@
+//go:build go1.18
+// +build go1.18
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package runtime
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/internal/exported"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/internal/shared"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/tracing"
+)
+
+const (
+	attrHTTPMethod     = "http.method"
```

Comment:
otel attributes might change (these ones have a very high chance to become stable very soon, but can change one more time before that).

To help us and users with this, we should populate [schema_url](https://github.com/open-telemetry/opentelemetry-go/blob/main/trace/config.go#L27)  when creating OTel tracer. 


Do we currently do it?

Reply:
We'd do this in our adapter.
```go
tracer := tracerProvider.Tracer(namespace,
	trace.WithInstrumentationVersion(version),
	trace.WithSchemaURL(semconv.SchemaURL),
)
```